libs/blocks/comparison-table/comparison-table.css

```diff
@@ -0,0 +1,490 @@
+  display: flex;
+}
+
+.comparison-table .sub-header-item-container .btn-container .action-area .con-button {
+  justify-content: center;
+}
+
+.comparison-table .sub-header-item-container:last-of-type {
+  border: none;
+  padding: 0;
+}
+
+.comparison-table .sub-header-item-container p:not(:has(picture, a)) {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  word-break: break-word;
+}
+
+.comparison-table .sub-header-item-container *:is(p, h1, h2, h3, h4, h5, h6) {
+  text-align: center;
+  word-break: break-word;
+  color: var(--color-black);
+}
+
+.comparison-table .sub-header-item-container *:is(h1, h2, h3, h4, h5, h6) {
+  font-size: var(--type-body-s-size);
+  font-style: normal;
+  font-weight: var(--type-detail-all-weight);
+  line-height: var(--type-body-s-lh);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.comparison-table .price {
+  font-size: var(--type-body-s-size);
+  font-style: normal;
+  font-weight: var(--type-detail-all-weight);
+  line-height: 125%;
+  color: var(--color-black);
+  display: flex;
+}
+
+.comparison-table .sub-header-item-container em {
+  font-size: var(--type-body-xxs-size);
+  font-weight: 400;
+  line-height: 125%;
+}
+
+.comparison-table .sub-header-item-container .description em a {
+  color: var(--color-black);
+}
+
+.comparison-table .price.price-strikethrough {
+  font-size: var(--type-body-xxs-size);
+}
+
+.comparison-table .header-item.hidden,
+.comparison-table .table-cell.hidden,
+.comparison-table .accessibility-header-cell.hidden {
+  display: none;
+}
+
+.comparison-table .accessibility-header-row {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border-width: 0;
+}
+
+.comparison-table .mobile-filter-select {
+  width: calc(100% - 2px);
+  height: 22px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url('./img/down-arrow-filter.svg');
+  background-color: transparent;
+  display: none;
+  cursor: pointer;
+  color: transparent;
+  position: absolute;
+  border: none;
+  border-top: 1px solid var(--comparison-border-color);
+  bottom: 1px;
+  border-end-start-radius: var(--comparison-border-radius);
+  border-end-end-radius: var(--comparison-border-radius);
+}
+
+.comparison-table .mobile-filter-select option {
+  color: var(--color-black);
+}
+
+.comparison-table .header-content.sticky .sub-header-item-container:first-child {
+  padding-bottom: 0;
+}
+
+.comparison-table .sub-header-item-container:first-child:not(:has(.mobile-filter-select)) {
+  padding-bottom: var(--spacing-xxs);
+}
+
+.comparison-table .table-container .table-column-header *:is(h1, h2, h3, h4, h5, h6) {
+  max-width: calc(100% - 36px);
+}
+
+[dir="rtl"] .comparison-table .price {
+  direction: ltr;
+}
+
+@media screen and (min-width: 600px) and (max-width: 899px) {
```

Comment:
Is there a reason for the range instead of mobile-first, then tablet, then desktop, etc. approach?